cli/cmd/alert_rules.go

```diff
@@ -0,0 +1,350 @@
+			if err != nil {
+				return errors.Wrap(err, "unable to get alert rules")
+			}
+			if len(alertRules.Data) == 0 {
+				msg := `There are no alert rules configured in your account.
+
+Get started by integrating your alert rules to manage alerting using the command:
+
+    $ lacework alert-rule create
+
+If you prefer to configure alert rules via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Settings > Alert Rules.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+			if cli.JSONOutput() {
+				return cli.OutputJSON(alertRules)
+			}
+
+			var rows [][]string
+			for _, rule := range alertRules.Data {
+				rows = append(rows, []string{rule.Guid, rule.Filter.Name, rule.Filter.Description, rule.Filter.Status()})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "DESCRIPTION", "ENABLED"}, rows))
+			return nil
+		},
+	}
+	// show command is used to retrieve a lacework alert rule by resource id
+	alertRulesShowCommand = &cobra.Command{
+		Use:   "show",
+		Short: "get an alert rule by id",
+		Long:  "Get a single alert rule by it's ID.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			var response api.AlertRuleResponse
+			err := cli.LwApi.V2.AlertRules.Get(args[0], &response)
+			if err != nil {
+				return errors.Wrap(err, "unable to get alert rule")
+			}
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(response)
+			}
+
+			alertRule := response.Data
+			var headers [][]string
+			headers = append(headers, []string{alertRule.Guid, alertRule.Filter.Name, alertRule.Filter.Description, alertRule.Filter.Status()})
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "DESCRIPTION", "ENABLED"}, headers))
+			cli.OutputHuman("\n")
+			cli.OutputHuman(buildAlertRuleDetailsTable(alertRule))
+
+			return nil
+		},
+	}
+
+	// delete command is used to remove a lacework alert rule by resource id
+	alertRulesDeleteCommand = &cobra.Command{
+		Use:   "delete",
+		Short: "delete a alert rule",
+		Long:  "Delete a single alert rule by it's ID.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			err := cli.LwApi.V2.AlertRules.Delete(args[0])
+			if err != nil {
+				return errors.Wrap(err, "unable to delete alert rule")
+			}
+			return nil
+		},
+	}
+
+	// create command is used to create a new lacework alert rule
+	alertRulesCreateCommand = &cobra.Command{
+		Use:   "create",
+		Short: "create a new alert rule",
+		Long:  "Creates a new single alert rule.",
+		RunE: func(_ *cobra.Command, args []string) error {
+			if !cli.InteractiveMode() {
+				return errors.New("interactive mode is disabled")
+			}
+
+			response, err := promptCreateAlertRule()
+			if err != nil {
+				return errors.Wrap(err, "unable to create alert rule")
+			}
+
+			cli.OutputHuman(fmt.Sprintf("The alert rule was created with ID %s \n", response.Data.Guid))
```

Comment:
in the table we call this column `GUID`, so update this or the table.
```suggestion
			cli.OutputHuman(fmt.Sprintf("The alert rule was created with GUID '%s'.\n", response.Data.Guid))
```